sdks/java/core/src/test/java/org/apache/beam/sdk/values/TypedPValueTest.java

```diff
@@ -77,14 +78,10 @@ public void testUntypedOutputTupleTagGivesActionableMessage() {
 
     thrown.expect(IllegalStateException.class);
     thrown.expectMessage("No Coder has been manually specified");
-    thrown.expectMessage(
-        containsString("Building a Coder using a registered CoderFactory failed"));
-    thrown.expectMessage(
-        containsString("Building a Coder from the @DefaultCoder annotation failed"));
-    thrown.expectMessage(
-        containsString("Building a Coder from the fallback CoderProvider failed"));
+    thrown.expectMessage("Building a Coder using a registered CoderFactory failed");
```

Comment:
As suggested, we can reduce the number of coder factories and list all their failure reasons. If users do enable many additional coder factories, then the error message will be very long.